client/packages/coldchain/src/Equipment/ListView/Toolbar.tsx

```diff
@@ -90,6 +90,10 @@ export const Toolbar = () => {
           label: t('status.not-in-use', { ns: 'coldchain' }),
           value: AssetLogStatusInput.NotInUse,
         },
+        {
+          label: t('status.unserviceable', { ns: 'coldchain' }),
```

Comment:
The `{ns: 'coldchain' }` doesn't actually seem to be doing anything here 🤷‍♂️

Reply:
We've moved all our translations into `common.json` so these `ns` parameters can be removed now.
Was relevant when things were split up more...